src/components/VideoPlayer/index.js

```diff
@@ -155,6 +153,33 @@ export default class VideoPlayer extends PureComponent {
     this.calculateFill()
   }
 
+  handleReady = () => {
+    const { hasAutoplay, isMuted, onVideoReady } = this.props
+
+    if (isMuted) {
+      this.video.muted(true)
```

Comment:
while we are passing `muted={isMuted}` in the render, there is evidence it is being stripped out by react

https://github.com/facebook/react/issues/6544

so we are programmatically muting here